source/includes/_webhooks.md

```diff
@@ -801,6 +801,31 @@ This webhook is fired when the system has successfully recorded a repayment requ
 This webhook is fired whenever the system marks a single repayment as cleared.
 I.e. the money movement for the singular investor repayment has occurred.
 
+## CASH_INTEREST_REPAYMENT_CLEARING
+```json
+{
+  "id": "d6adb525-adac-4cc0-a86f-a9c0e68a7bb8",
+  "type": "CASH_INTEREST_REPAYMENT_CLEARING",
+  "dateTime": "2021-11-26T21:51:32.574621Z",
+  "content": {
+    "repaymentBatchReference": "REPAYMENT-2b86a1bb-fe9d-4476-9e6b-0e29477e251d",
+    "repaymentId": "REPAYMENT_ID",
+    "amount": {
+      "amount": 10,
+      "currency": "GBP"
+    },
+    "type": "CASH_INTEREST",
+    "status": "CLEARED",
+    "postedAt": "2021-11-26T21:51:32.542533Z",
+    "clearedAt": "2021-11-26T21:51:32.543340Z",
+    "investorId": "20be5b14-bf13-4969-9aea-8adedbbc72c6",
```

Comment:
So... this is me catching it late _but_  I think this requires a change in payments code & then an update here. We never communicate out `investorId`(s) like this to platforms so they may get confused. 
Instead, we populate the clientId on these webhooks of the form e.g. `SPEC-12345`.

Reply:
Thanks, @victokoh! I forgot to remove one of the @JsonProperty("investorId") annotations, it is actually displaying the clientId though.